### DISCUSSION TITLE
Help: Devices partially resetting

### DISCUSSION THREAD

**Discussion by g0lgs** [arendst/Tasmota#11929] [Q&A] [answered]
I have a few Tasmota converted power strips that seem to randomly revert to Sonoff Basic GPIO configuration - I suspect that this happens when the WiFi network to which they are connected randomly drops the connection a few times over a short period of time. 

To get them operating normally I have to access them using the Web interface (having found the correct IPs) and go to Configuration -> Configure Other and re-active my template.

Is there something I need to change in the configuration to stop this happening ?

**Comment by sfromis** [comment, marked as answer]
Resetting to Sonoff Basic is usually because of some serious error causing a boot loop. To avoid a bricked device, Tasmota will instead reset config details till able to boot successfully. Should not be related to wifi dropouts.

**Reply by g0lgs** [reply]
As they have been working for a while I can't see any other reason why they would loose the settings. The location they are at does seem to suffer from WiFi or Internet drop-outs and my only thought it is somehow related to those.

**Reply by ascillato** [reply]
Power drop outs will trigger the boot loop protection. You can disable it by setoptions 36 and 65. Please, check the docs on how to use those commands.

**Reply by g0lgs** [reply]
I don't think it is power drop-outs either as nothing else seems to be affected - various RPI's and other equipment has not lost power or been rebooted, but I can see in the logs that some of them have lost and re-established WiFi connections.

I will try setoption 36, but as it mostly only happens only every few weeks I may not see if it has helped for a while.

I don't really want to change setoption65.

**Comment by g0lgs** [comment]
Further to this it happened again a short time ago, on checking I found intermittent WiFi connectivity for a number devices, which includes various user devices (workstations etc) as well as the MQTT server that the Tasmota devices are using. A restart of the MQTT server (running on a Raspberry Pi) appears to have got the WiFi back to normal, but I had to re-activate the templates on the affected Tasmota devices.

This was with setoption36 set to 10 on the affected Tasmota devices. 

I think I will now try setoption36 at 0
